selenium/src/test/java/au/org/emii/portal/tests/BrowserStackTest.java

```diff
@@ -0,0 +1,19 @@
+package au.org.emii.portal.tests;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class BrowserStackTest extends BaseTest {
+
+    @Test(invocationCount = invocationCount, threadPoolSize = threadPoolSize)
+    public void BrowserStackTest() {
+        getDriver().get("http://www.google.com");
```

Comment:
Just seems cleaner to me using a general utility class throughtout rather than using the lower level collaborators but its not a biggie.